<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plotWidget</class>
 <widget class="QMainWindow" name="plotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="plot_canvas" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>系统</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>绘图</string>
    </property>
    <addaction name="action_animation_1"/>
    <addaction name="action_animation_2"/>
    <addaction name="separator"/>
    <addaction name="action_normalplot_1"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_animation_1">
   <property name="text">
    <string>动态图表(1)</string>
   </property>
  </action>
  <action name="action_animation_2">
   <property name="text">
    <string>动态图表(2)</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_normalplot_1">
   <property name="text">
    <string>普通绘图(1)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
